### DISCUSSION TITLE
Allure Report Extension on Azure Devops Fails to Load Report

### DISCUSSION THREAD

**Discussion by Novvator** [allure-framework/allure2#2815] [Bug Reports] [answered]
Allure Extension Version 1.4.3
Since today 28/11/2024 Allure Report tab in Azure Devops started failing to load reports, even from pipeline runs of previous days.
The tab can be opened as expected, it shows message "Build is still in progress" then "Loading available reports" and then it becomes blank white. You can see the attached screeshots. No updates were made on our end to the allure reports, they were working as expected until today.

<img width="899" alt="1" src="https://github.com/user-attachments/assets/640b14ce-2987-46c2-a728-880f1041bfb6">
<img width="748" alt="2" src="https://github.com/user-attachments/assets/b94e6577-8200-45d2-b90f-e4d582e8df3e">
<img width="744" alt="Screenshot_11" src="https://github.com/user-attachments/assets/de8838af-49d0-4b22-8554-b0d3262a20ba">

**Comment by fourdr-r** [comment]
Hello,

We are having the exact same issue. Note that downloading the report from Azure DevOps and opening it locally seems to work as expected.

**Reply by baev** [reply]
Please update to the version 1.4.4

**Comment by dhansak79** [comment]
I am also experiencing this issue.

**Comment by dhansak79** [comment]
Is there a way to roll back to the previous version in ADO? I've spent the last few weeks getting a new project setup and have a demo at 3pm today - rotten timing!

**Reply by epszaw** [reply]
Hello, I'll do rollback right now to make possible to make the demo for you 🙏

**Reply by vladide** [reply]
Could you please confirm, if this is something central in ADO that you would rollback to fix it?

**Reply by epszaw** [reply, marked as answer]
Yes, versions 1.4.4 is a previous version. You should be able to use it right now

**Reply by epszaw** [reply]
@dhansak79 could you validate that everything works now in your case?

**Reply by stevik** [reply]
@epszaw I can confirm that version 1.4.4 is working correctly. Thanks for the prompt action!

**Reply by fourdr-r** [reply]
I just tried on my end too when I saw your replies here and I can confirm using the 1.4.4 Extension with Allure Reports' default version (2.27.0) works!
![image](https://github.com/user-attachments/assets/756cead8-b3a2-4d90-b0e7-427c303f8b40)

**Reply by epszaw** [reply]
Glad to help you, guys!

**Comment by stevik** [comment]
I am observing the same behavior after updating to version 1.4.3 (it was working without issues on version 1.4.2).

I tested it on the latest available version of Google Chrome (Version 131.0.6778.86 (Official Build) (arm64)), also on Safari Version 18.0 (20619.1.26.31.6) and Firefox 133.0 (aarch64) 
I am using Allure version 2.30.0 and macOS Sequoia 15.0 (not sure if this is relevant)

I checked the browser console and the network tab, but I don't see any errors there, only something about third party-cookies —see the attached video.


https://github.com/user-attachments/assets/0a19ab94-c9ca-4644-a570-1d2496a95dff

**Reply by baev** [reply]
Please update to the version 1.4.4

**Reply by stevik** [reply]
Thanks, 1.4.4 is working correctly.

**Comment by petch-dan** [comment]
I'm also experiencing this, updated all our dependencies to latest versions, etc. but same as others.

**Reply by baev** [reply]
Please update to the version 1.4.4

**Reply by petch-dan** [reply]
Thanks for rolling back so quick, all working now!

**Comment by dhansak79** [comment]
My reports are working now! Well done for being so responsive - I am genuinely impressed at how quickly you dealt with this.  Definitely not the norm!  Thank you so much!

**Reply by fourdr-r** [reply]
100% agree, it should not be overlooked! Kudos @epszaw :)

**Reply by jyothish-aveva** [reply]
I am seeing this issue again, which seems working before. I downloaded the report directory and verified in local. Reports are getting generated in local.
<img width="2100" height="1511" alt="Screenshot 2025-12-18 144958" src="https://github.com/user-attachments/assets/0b52724f-20dc-4de8-b69d-6a9d0cbe899f" />

**Reply by ymouhat** [reply]
HI  @jyothish-aveva   yes it started to be reported here https://github.com/orgs/allure-framework/discussions/3198
